### ISSUE TITLE
Console output disappears on Terminal-Normal mode after switching from other buffer

### ISSUE THREAD

**Issue by igrep** [vim/vim#4167]
# Steps to reproduce

1. Launch vim.
    ```
    $ vim -u NONE
    ```
1. Open a terminal buffer by some command.
    ```
    :terminal ++curwin bash
    ```
1. Press `<C-\><C-n>` to escape to Terminal-Normal mode.
1. `:set bufhidden=hide` to prevent the buffer from getting deleted.
1. `:e some-file` to switch to another buffer.
1. `:bn` to get back to the terminal buffer opened before.
1. Press `i` to switch to Terminal-Job mode again.
1. Then press `<C-\><C-n>` to escape to Terminal-Normal mode again.

# Actual result

The output of the terminal buffer is cleared up.
Nothing can be copied.
Pressing `i` again to switch to Terminal-Job mode shows the output again.

# Expected result

The output of the terminal buffer is kept.
It can be copied by the yank commands.

# Screencast

![GIF](https://user-images.githubusercontent.com/227057/54899396-106c0b80-4f13-11e9-9c51-5ecc5e97958c.gif)

# My environment

The result of `:version` (excerpt):

```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Mar 17 2019 01:24:47)
MS-Windows 64 ビット コンソール 版
適用済パッチ: 1-1010
```

**Comment by k-takata** [comment]
I think this is the same issue as #4126, and fixed by 8.1.1018.
(Cannot reproduce on 8.1.1048.)

**Comment by igrep** [comment]
Oh, sorry for filing a duplicate issue. I'll close.
